cms/static/cms/js/modules/cms.structureboard.js

```diff
@@ -112,6 +112,12 @@ $(document).ready(function () {
 				}
 			});
 
+			//fix issue #4407
+			$('.cms-toolbar-expanded .cms_dragarea .cms_draggables .cms_draggable').bind('cms.structureboard.menustate.change', function(e){
```

Comment:
there's definitely no need for `.cms-toolbar-expanded` as this only indicates if toolbar is hidden or not. 
maybe `.cms_draggable` is enough?